src/client/auth.go

```diff
@@ -11,6 +11,10 @@ func (c APIClient) IsAuthActive() (bool, error) {
 		Resource: &auth.Resource{Type: auth.ResourceType_CLUSTER},
 	})
 	switch {
+	case err == nil:
+		return true, nil
+	case auth.IsErrNotAuthorized(err):
+		return true, nil
```

Comment:
This method wasn't working in the case where nothing bad happened, which blocked deploying the IDE.